tools/astarte_export/config/runtime.exs

```diff
@@ -0,0 +1,4 @@
+import Config
+
+config :cqerl,
+  cassandra_nodes: [{System.get_env("CASSANDRA_DB_HOST"), System.get_env("CASSANDRA_DB_PORT")}]
```

Comment:
Xandra, too, should receive the info on nodes and port (see https://hexdocs.pm/xandra/0.13.1/Xandra.html#start_link/1-options and https://github.com/astarte-platform/astarte_data_access/blob/45d4d20ae662751f47f4b8f2f9d5e302d5485ea9/lib/astarte_data_access/config.ex#L158-L168)